### DISCUSSION TITLE
Could not find com.github.Minestom:Minestom:529_extension_improvement-SNAPSHOT.

### DISCUSSION THREAD

**Discussion by amrouda50** [Minestom/Minestom#1377] [Help] [answered]
I suddenly got the following compilation error   , although it was working before :

Execution failed for task ':compileJava'.
> Could not resolve all files for configuration ':compileClasspath'.
   > Could not find com.github.Minestom:Minestom:529_extension_improvement-SNAPSHOT.
     Searched in the following locations:
       - https://repo.maven.apache.org/maven2/com/github/Minestom/Minestom/529_extension_improvement-SNAPSHOT/maven-metadata.xml
       - https://repo.maven.apache.org/maven2/com/github/Minestom/Minestom/529_extension_improvement-SNAPSHOT/Minestom-529_extension_improvement-SNAPSHOT.pom
       - https://jitpack.io/com/github/Minestom/Minestom/529_extension_improvement-SNAPSHOT/maven-metadata.xml
       - https://jitpack.io/com/github/Minestom/Minestom/529_extension_improvement-SNAPSHOT/Minestom-529_extension_improvement-4eec3d10a3-1.pom
     Required by:
         project :

Possible solution:
 - Declare repository providing the artifact, see the documentation at https://docs.gradle.org/current/userguide/declaring_repositories.html

**Comment by KorneIlles** [comment]
outdated mienstom version

**Comment by sarpedondev** [comment, marked as answer]
Jitpack is broken with branch-SNAPSHOT. Use branch-COMMIT_ID or just COMMIT-ID as version.

**Comment by sarpedondev** [comment]
Else you could follow this message from the FAQ:

**Getting this error while resolving dependencies?**
`Could not resolve com.github.Minestom:Minestom:someversion`
Add this to your build.gradle:
```gradle
java {
    toolchain {
        languageVersion = JavaLanguageVersion.of(17)
    }
}
```

Source: https://discord.com/channels/706185253441634317/757676113886576800/905500990793461781
